autowrap/ConversionProvider.py

```diff
@@ -1060,6 +1164,14 @@ def matches(self, cpp_type):
     def matching_python_type(self, cpp_type):
         return "list"
 
+    def matching_python_type_full(self, cpp_type):
+        tt, = cpp_type.template_args
+        try: # TODO if the template type is a typedef we currently have a problem
+            inner_conv = self.converters.get(tt)
+            return "List[%s]" % inner_conv.matching_python_type_full(tt)
+        except:
+            return "Any"
```

Comment:
does this handle the template case? the comment "have a problem" could be improved

Reply:
seems to work